doc/tools/make_rst.py

```diff
@@ -1721,17 +1778,19 @@ def format_text_block(
                     tag_text = "\n::\n"
 
                 inside_code = True
-                inside_code_tag = cmd
+                inside_code_tag = tag_state.name
+                ignore_code_warnings = "safe" in tag_state.arguments
```

Comment:
As discussed on RC, I would suggest using `nolint`, or another option along those lines. Could also be `nowarn` or `skip-lint` or something like this.

`safe` seems a bit weird to me in the context of `code` tags.

Reply:
I like `skip-lint`, will update to it.